docker/containers.sls

```diff
@@ -11,7 +11,7 @@ docker-image-{{ name }}:
       - service: docker-service
 
 {# TODO: SysV init script #}
-{%- set init_system = salt["cmd.run"]("ps -p1 | grep -q systemd && echo systemd || echo upstart") %}
+{%- set init_system = salt["cmd.run"]("ps -p1 | grep -q systemd && echo systemd || echo upstart", python_shell=True) %}
```

Comment:
Modern Salt versions provide a couple of nice alternatives to this:
```jinja
{%- set init_system = grains['init'] %}
{# or #}
{%- set init_system = salt['test.provider']('service') %}
```

Reply:
This change fixes the immediate problem.  I'd be happy to submit another PR later that refactors the code in question.

Reply:
The Clean Code defines so-called "The Boy Scout Rule": Leave the campground cleaner than you found it.
If something could be done in the right way, it should be immediately done so with no intermediate steps.
But that's up to you and the formula maintainers, of course. This is only my opinion and it's good to make any valuable contribution. Thanks.